docs/en/setup/backend/backend-cluster.md

```diff
@@ -73,6 +73,18 @@ cluster:
     hostPort: ${SW_CLUSTER_CONSUL_HOST_PORT:localhost:8500}
 ```
 
+## Nacos
```

Comment:
Why do you add document inside `Consul` paragraph?

Reply:
consul contains hostPort and serviceName, nacos also contains hostPort and serviceName,so i just copy it and change the cluster Name and default port

Reply:
I am talking about the document, the following document is part of Consul document, if your Nacos document needs this too, please copy it in your paragraph. 
> Same as Zookeeper coordinator, in some cases, oap default gRPC host and port in core are not suitable for internal communication among the oap nodes. The following setting are provided to set the hot and port manually, based on your own LAN env.
internalComHost, the host registered and other oap node use this to communicate with current node.
internalComPort, the port registered and other oap node use this to communicate with current node.

Right now, it looks like Consule doesn't support his, see at https://github.com/apache/skywalking/blob/dc39825a74ad31df0c1ab2736d3fed4a2f31382d/docs/en/setup/backend/backend-cluster.md#consul

Reply:
`internalComHost` and `internalComPort` things are not supported in Nacos. Suggest to add.

Reply:
oo..its my fault.
I have not noticed that.
i'll fix